tests/test_indexutils.py

```diff
@@ -41,8 +44,21 @@ def setUp(self):
         tarball.extractall(self.index_dir)
         tarball.close()
 
-        self.searcher = pysearch.SimpleSearcher(f'{self.index_dir}lucene-index.cacm')
-        self.index_utils = pyutils.IndexReaderUtils(f'{self.index_dir}lucene-index.cacm')
+        self.index_path = os.path.join(self.index_dir, 'lucene-index.cacm')
+        self.searcher = pysearch.SimpleSearcher(self.index_path)
+        self.index_utils = pyutils.IndexReaderUtils(self.index_path)
+
+    def test_tfidf_vectorizer(self):
+        vectorizer = TfidfVectorizer(self.index_path, min_df=5)
+        train_docs = ['CACM-0239', 'CACM-0440', 'CACM-3168', 'CACM-3169']
+        train_labels = [1, 1, 0, 0]
+        test_docs = ['CACM-0634', 'CACM-3134']
+        train_vectors = vectorizer.get_vectors(train_docs)
+        test_vectors = vectorizer.get_vectors(test_docs)
+        clf = MultinomialNB()
+        clf.fit(train_vectors, train_labels)
+        pred = clf.predict(test_vectors)
+        self.assertListEqual([1, 0], list(pred))
```

Comment:
Can you add an assertion of the actual classifier score?
This will defend against underlying sklearn changes.

Reply:
done